observations/forward_operators/obs_def_rttov_mod.rst

```diff
@@ -6,51 +6,39 @@ MODULE ``obs_def_rttov_mod``
 Overview
 --------
 
-The DART RTTOV observation module, including the observation operators for the two primary 
+The DART-RTTOV observation module includes observation operators for the two primary 
 RTTOV-observation types -- visible/infrared radiances and microwave 
-radiances/brightness temperatures.
-
-DART can be built with either RTTOV v12 *or* v13. Edit :ref:`&preprocess_nml <preprocess>` to select
-the appropriate obs_def:
-
-- obs_def_rttov_mod.f90 for v12.3
-- obs_def_rttov13_mod.f90 for v13 (contributed by Lukas Kugler of the University of Vienna).  
-
-Note the namelist options for &obs_def_rttov_nml differ for v12 and v13.
-
-- `RTTOV v12 Namelist`_ &obs_def_rttov_nml
-- `RTTOV v13 Namelist`_ &obs_def_rttov_nml
```

Comment:
How to compile with rttov v12 or v13 seems to be missing from the new documentation, unless I'm missing something. This seems pretty important info for compiling DART with RTTOV.